src/components/application_manager/src/hmi_command_factory.cc

```diff
@@ -157,6 +159,15 @@
 #include "application_manager/commands/hmi/on_app_permission_changed_notification.h"
 #include "application_manager/commands/hmi/on_event_changed_notification.h"
 
+// Remote Control
+#include "application_manager/commands/hmi/rc_get_interior_vehicle_data_capabilities_request.h"
```

Comment:
@LuxoftAKutsan, @okozlovlux, perhaps it should be included only if `SDL_REMOTE_CONTROL` defined